buildpacks/npm/lib/build.sh

```diff
@@ -49,6 +49,8 @@ clear_cache_on_stack_change() {
 	fi
 
 	if [[ ! -f "${layers_dir}/store.toml" ]]; then
+		# shellcheck disable=SC2005
+		curr_node_version="$(echo "$(node -v)")"
```

Comment:
Do you still need the if statement from 52 - 59 if you have the new `write_to_store_toml` function?